### PR TITLE
Add support for Ruby 2.7 and drop for 2.4

### PR DESCRIPTION
**Pull request by Pontus4**

### CODE REVIEW

File: twingly-url_cache.gemspec [hunk 1]
```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "dalli", "~> 2.7"
   spec.add_dependency "retryable", "~> 2.0"
 
-  spec.add_development_dependency "bundler", "~> 1.10"
+  spec.add_development_dependency "bundler", ">= 1.10", "< 2.2"
```
Comment (walro) [inline twingly-url_cache.gemspec:22]: not sure if viable, but if we drop support for 2.5 too I don't think we need to bother about bundler here, as we can count on everyone having some version of bundler installed

Comment (walro) [reply twingly-url_cache.gemspec:22]: For fun I actually tried creating a new gem using `bundler gem <NAME> -t` and looked at the output. It seems like no requirements are added to the gemspec (not even bundler itself) so I wonder if this is some old copy pasta from some old gem best practices. :) The initial commit in this repo does not really tell.

Comment (Pontus4) [reply twingly-url_cache.gemspec]: We try to drop support for Ruby 2.5 and and remove the bundler dependency then :)

Comment (Pontus4) [reply twingly-url_cache.gemspec]: aaand it worked 👍

**Comment by walro** [comment]
> Remove support for Ruby 2.5

The commit message could be helped by a why :)

**Comment by Pontus4** [comment]
> > Remove support for Ruby 2.5
> 
> The commit message could be helped by a why :)

I extended it a bit :-)

**Review by roback — APPROVED** [review]
Changes LGTM!

I don't think we need to release a new version of this gem right now since we didn't have to do any "real" changes here.

**Comment by walro** [comment]
> I don't think we need to release a new version of this gem right now since we didn't have to do any "real" changes here.

One can take a look at https://github.com/twingly/twingly-url_cache/compare/v2.0.0...master to see if there's something else we want released though (and no, there isn't :))

**Pull request merged**